<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电池缺陷检测软件</string>
  </property>
  <property name="windowIcon">
   <iconset resource="batguard.qrc">
    <normaloff>:/ico/logo</normaloff>:/ico/logo</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>缺陷检测阈值</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="verticalSlider_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>AR PL UKai CN</family>
          <pointsize>36</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 101, 164);
color: rgb(238, 238, 236);</string>
        </property>
        <property name="text">
         <string>电池缺陷检测软件</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>二值化阈值</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="verticalSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>240</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCheck"/>
   <addaction name="actionCheckTop"/>
   <addaction name="actionCheckCover"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="batguard.qrc">
     <normaloff>:/ico/stop</normaloff>:/ico/stop</iconset>
   </property>
   <property name="text">
    <string>停止检测</string>
   </property>
   <property name="toolTip">
    <string>停止检测</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="batguard.qrc">
     <normaloff>:/ico/start</normaloff>:/ico/start</iconset>
   </property>
   <property name="text">
    <string>开始检测</string>
   </property>
   <property name="toolTip">
    <string>开始检测</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="icon">
    <iconset resource="batguard.qrc">
     <normaloff>:/ico/check</normaloff>:/ico/check</iconset>
   </property>
   <property name="text">
    <string>划痕检测</string>
   </property>
   <property name="toolTip">
    <string>划痕检测</string>
   </property>
  </action>
  <action name="actionCheckTop">
   <property name="text">
    <string>顶盖检测</string>
   </property>
   <property name="toolTip">
    <string>顶盖检测</string>
   </property>
  </action>
  <action name="actionCheckCover">
   <property name="text">
    <string>包覆破损</string>
   </property>
   <property name="toolTip">
    <string>包覆破损</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="batguard.qrc"/>
 </resources>
 <connections/>
</ui>
